### PR TITLE
space-travelers final work

### PR DESCRIPTION
**Pull request by AbrahaKahsay**
# Projection description
In This project I and my coding partner have implemented:
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write unit tests with React Testing Library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.
- Perform a code review for a team member.
## Technologies used
- HTML, CSS, JavaScript, React, Redux, Webpack, Babel
- Github, Git, Linters, VS Code
- Chrome Browser

### CODE REVIEW

File: src/components/tests/Profile.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    const { container } = render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Profile />
+        </Provider>
+      </MemoryRouter>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+});
```
Comment (rotimiazeez) [inline src/components/tests/Profile.test.js:19]: - Some of your tests are failing, kindly fix them 👍🏽
Screenshot from 2022-08-27 01-50-23
Screenshot from 2022-08-27 01-50-58
Screenshot from 2022-08-27 01-51-21

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯
- Good UI 🎭
- Descriptive README 📖
- Good commit messages 👍🏽
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbrahaKahsay** [comment]
Dear @rotimiazeez thanks for the review and the catch on the errors. Now the issue is fixed.

File: src/components/tests/Profile.test.js (hunk 1 above)
Comment (peterrobert) [reply src/components/tests/Profile.test.js:19]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @AbrahaKahsay, 

**Highlights**

- [ ] Great work making all the requested changes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbrahaKahsay** [comment]
@peterrobert Thanks for the review! Have great weekend 😃

**Pull request merged**